tools/gulp/tasks/payload.ts

```diff
@@ -93,10 +93,10 @@ async function calculatePayloadDiff(database: firebaseAdmin.database.Database, c
 
   // Calculate the payload diffs by subtracting the previous size of the FESM ES2015 bundles.
   const cdkFullSize = currentPayload.cdk_fesm_2015;
-  const cdkDiff = cdkFullSize - previousPayload.cdk_fesm_2015;
+  const cdkDiff = roundFileSize(cdkFullSize - previousPayload.cdk_fesm_2015);
```

Comment:
Why not just use `toFixed`?

Reply:
It returns a string instead of a number. I don't think re-parsing is better.

Reply:
Generally you want to keep your values in number format as long as possible and them format them just before or past of displaying them.

Reply:
Yeah that's what this does now. `toFixed()` returns a string instead of a number. The formatting will happen in the Firebase function.